app/views/order_items/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2>Your picks</h2>
+  <% @order_items.each do |item| %>
+    <div class="addedFoodDiv"> 
+    <%= item.item.name %>
+    <%= item.item.description %>
+    <%= item.item.price %> 
+    </div>
+    <% @total_price += item.item.price %>
```

Comment:
Please avoid doing calculations in a view. If needed, please extract it to a helper method. In this case, though, you don't have to do this calculation. If you examine the documentation of the acts_as_shopping_cart, you'll see that there are methods to get a total of the order. You can do even more, like calculating taxes and so on....

Reply:
Moved it out of the view to a helper method. Built in methods gave me such strange figures.

Reply:
How so?